scripts/clang-format.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+if test x"$1" = x; then 
+	FILES=`git ls-files | grep '\.[ch]$'`
+fi
+if test x"$1" = xchanged; then 
+	FILES=`git diff --name-only | grep '\.[ch]$'`
+fi
+
+for i in $FILES; do
+	clang-format-7 $i > /tmp/tmp-clang.c
```

Comment:
Why `clang-format-7` instead of just `clang-format`? On my local Ubuntu 16.04 system, `clang-format --version` reports `3.8.0`. Are there really that many versions between?